chianft/util/mint.py

```diff
@@ -202,6 +203,8 @@ async def submit_spend_bundles(
         # make sure we have a dir for offers if needed
         if create_sell_offer:
             Path("offers").mkdir(parents=True, exist_ok=True)
+        if list_coins:
+            Path("coins_list").mkdir(parents=True, exist_ok=True)
```

Comment:
I'm not sure we need to create a directory since it's just creating one file. Users could enter the full path along with the file name in the -lc option. So maybe this line should only call mkdir if there is a path with the filename.